src/services/filtredTodos.tsx

```diff
@@ -0,0 +1,12 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export function getFilteredTodos(todos:Todo[], filter: Filter) {
+  return todos.filter(
+    todo => todo.title
+      .toLowerCase().includes(filter.title.toLowerCase())
+  && (filter.status === 'all'
+  || (filter.status === 'active' && !todo.completed)
+  || (filter.status === 'completed' && todo.completed)),
+  );
```

Comment:
in this place has bad readability. Using chaining* with || and && is not a good idea. 
The best way use switch or if`s for better readability.